src/app/components/content/IsaacQuestion.tsx

```diff
@@ -62,20 +57,9 @@ export const IsaacQuestion = withRouter(({doc, location}: {doc: ApiTypes.Questio
         !!locked
     );
 
-    const tooManySigFigsFeedback = <p>
```

Comment:
Are significant figures irrelevant to CS? Aside from symbolic chemistry the supported question types should be the same, I would have thought?